src/components/Dashboard/Carousel/Carousel.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+export const Carousel = ({ children, id, color }) => {
+  return (
+    <div className="dp-carousel" id={`carousel${id}`}>
+      <div className={`dp-carousel-wrapper dp-carousel-${color}`}>
+        <div className="dp-carousel-content">{children}</div>
+      </div>
+      <div className="dp-carousel-dots">
+        <input className="dp-carousel-dot" type="radio" value="first" id="first" name="carousel1" />
+        <input
+          className="dp-carousel-dot"
+          type="radio"
+          value="second"
+          id="second"
+          name="carousel1"
+        />
```

Comment:
Here you will have 2 radios always. And with the name carousel1 always, even if they are different instances of carousel component with different id. Perhaps you could add something pending on the id, to make the name like `name="{'radio'+id"}`
```suggestion
        <input className="dp-carousel-dot" type="radio" value="first" id="first" name={"radio"+id} />
        <input
          className="dp-carousel-dot"
          type="radio"
          value="second"
          id="second"
          name={"radio"+id}
        />
```